src/Blocks/Footnote/FootnotesBlockView.jsx

```diff
@@ -20,18 +20,10 @@ const alphabet = 'abcdefghijklmnopqrstuvwxyz';
 const FootnotesBlockView = (props) => {
   const { data, properties } = props;
   const { title, global, placeholder = 'Footnotes' } = data;
-  const [notesObj, setNodesObjs] = useState(null);
   const metadata = props.metadata ? props.metadata : properties;
-
-  useEffect(() => {
-    if (properties) {
-      const globalMetadata = global ? metadata : properties;
-      const blocks = getAllBlocksAndSlateFields(globalMetadata);
-      const notesObjResult = makeFootnoteListOfUniqueItems(blocks);
-
-      setNodesObjs(notesObjResult);
-    }
-  }, [properties]); // eslint-disable-line
+  const globalMetadata = global ? metadata : properties;
```

Comment:
@dobri1408 Please make sure that the removal of *useEffect* doesn't affect the copy & paste *text / blocks* with footnotes / zotero introduced by https://github.com/eea/volto-slate-footnote/commit/62e322825ad86bb992bfddfb1c5d894db48cf0dd